numpy/random/_examples/cython/extending_distributions.pyx

```diff
@@ -10,6 +10,8 @@ from cpython.pycapsule cimport PyCapsule_IsValid, PyCapsule_GetPointer
 from libc.stdint cimport uint16_t, uint64_t
 from numpy.random cimport bitgen_t
 from numpy.random import PCG64
+from numpy.random.c_distributions cimport (random_standard_uniform_fill,
+                                         random_standard_uniform_fill_f)
```

Comment:
Would probably just move both to the new line to keep things sanely indented?

Reply:
fixing